src/Slim/Middleware/Session.php

```diff
@@ -104,6 +104,8 @@ protected function startSession()
 
         session_name($name);
         session_cache_limiter(false);
-        session_start();
+        if ( session_status() == PHP_SESSION_NONE ) {
```

Comment:
When contributing to a project, it's nice to conform the code style the author is using. This bit of code works, but to standarize it, you could remove the spaces at the begining and end of the "if". :+1: